consoleApp/src/main/java/by/storeApp/StoreApp.java

```diff
@@ -0,0 +1,15 @@
+package by.storeApp;
+
+import by.store.RandomStorePopulator;
+import by.store.Store;
+
+import java.lang.reflect.InvocationTargetException;
+
+public class StoreApp {
+
+    public static void main(String[] args) throws InvocationTargetException, NoSuchMethodException, InstantiationException, IllegalAccessException {
+        RandomStorePopulator randomStorePopulator = new RandomStorePopulator();
+        Store store = randomStorePopulator.generateStore();
```

Comment:
generating store from populator is not the best choice. please think one more time (may be relay on my drawings from prev call) and refactor that.

Reply:
updated